org.mixedrealitytoolkit.spatialmanipulation/Solvers/RadialViewDisplayInitializeHelper.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+namespace MixedReality.Toolkit.SpatialManipulation
+{
+    /// <summary>
+    /// Makes a game object appear having a RadialView component in the center of the view,
+    /// at a distance equal to the average of the MinDistance and MaxDistance of the RadialView.
+    /// This prevents the object from 'racing into your view' and/or appear slanted
```

Comment:
remove single questions around "racing into you view"

Reply:
Try this instead.

```cs
/// <summary>
/// Positions a game object with a RadialView component at the center of the view,
/// maintaining a distance equal to the average of the MinDistance and MaxDistance 
/// properties of the RadialView. This prevents the object from rapidly approaching the 
/// viewer and ensures it appears upright.
/// </summary>
```

Reply:
Done. Thanks for this ready-to-copy header :)